### DISCUSSION TITLE
Kitty not automatically scrolling when changing window size

### DISCUSSION THREAD

**Discussion by b1two** [kovidgoyal/kitty#3297] [Q&A] [answered]
Hello,

First of all, thanks for the nice work !
I wonder if this is an expected/wanted behavior but I find the fullscreen/enlarging behavior a bit odd.
When a window is enlarged, it does not scroll the text to the bottom of the new space which forces the user to scroll manually. Is this something wanted ? I often enlarge/fullscreen kitty to see more information and I would prefer having the window scrolling automatically the text to the bottom in order to use all the space available.
Few screenshots to illustrate.

Initial window:
![kitty_normal-1612806462](https://user-images.githubusercontent.com/43686727/107260489-73e77f80-6a3e-11eb-964e-5f86c44db87d.png)

Enlarged window:
![kitty_enlarged-1612806572](https://user-images.githubusercontent.com/43686727/107260631-9a0d1f80-6a3e-11eb-8816-eac3a88c15ad.png)

Expected enlarged window:
![kitty_enlarged_good-1612806588](https://user-images.githubusercontent.com/43686727/107260653-a1342d80-6a3e-11eb-98d6-2847222c45b3.png)

I browsed the docs and could not find an option to do so, am I missing something ?

Thank you very much

**Comment by kovidgoyal** [comment, marked as answer]
It's by design, I find window contents changing on resizing to be very disorienting. As for having an option to do it, sure patches are welcome.

**Reply by b1two** [reply]
Do you know if there are any other people interested in such an option ? If so, I would gladly look into it, otherwise I will probably just get used to the current behavior.

**Comment by kovidgoyal** [comment]
I think one or two people have asked about it. If you search in issues you might find a reference.
